### ISSUE TITLE
ECE-Tools release notes are hard to find - Feedback on page: /guides/v2.3/cloud/release-notes/cloud-tools.html

### ISSUE THREAD

**Issue by dougkm** [magento/devdocs#3594]
# General issue

ECE-Tools release notes are not easy to find.  Can they be linked from the Release information section here - https://magento.com/technical-resources and here - https://devdocs.magento.com/guides/v2.3/release-notes/bk-release-notes.html?

-->

**Comment by dougkm** [comment]
I did eventually find that link but only after looking in several other locations and falling back to find it via google.  

Separate issue but note that the top hit for "Magento Cloud release notes" goes to this stub page rather than any actual release notes - https://devdocs.magento.com/guides/v2.3/cloud/release-notes/CloudReleaseNotes.html.

**Comment by jeff-matthews** [comment]
Agh... that page shouldn't exist. 

@hguthrie & @meker12, please remove the `guides/v2.3/cloud/release-notes/CloudReleaseNotes.md`  file.
